client/app/src/main/AndroidManifest.xml

```diff
@@ -20,20 +20,28 @@
             android:name="com.google.android.gms.vision.DEPENDENCIES"
             android:value="barcode" />
 
-        <activity android:name=".Activities.ScanActivity" />
-        <activity android:name=".Activities.SomethingWentWrong" />
-        <activity android:name=".Activities.CameraActivity"/>
-        <activity android:name=".Activities.NonOwnerBookViewActivity"/>
-        <activity android:name=".Activities.LoginActivity">
+        <activity android:name=".Activities.ScanActivity"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.SomethingWentWrong"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.CameraActivity"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.NonOwnerBookViewActivity"
+            android:windowSoftInputMode="adjustPan"/>
```

Comment:
there's no keyboard input in any of the above activities fyi, up to you if you want to remove them